csharp/Platform.Ranges/Range[T].cs

```diff
@@ -58,89 +70,144 @@ public Range(T minimum, T maximum)
         }
 
         /// <summary>
-        /// <para>Presents the Range in readable format.</para>
-        /// <para>Представляет диапазон в удобном для чтения формате.</para>
+        /// <para>Presents the <see cref="Range{T}"/> in readable format.</para>
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.object.tostring?view=net-5.0 - here you can find a better way to write this comment.